search/scorer/scorer_knn.go

```diff
@@ -60,6 +60,9 @@ func NewKNNQueryScorer(queryVector []float32, queryField string, queryBoost floa
 	}
 }
 
+// TODO: Better value needed here?
+const maxEuclideanDistance = 10000.0
```

Comment:
Gotta update this based on our conversation.

Reply:
yea for some reason the score seems to be 0.0 for doc product also so what to do then?

Reply:
Just to confirm, since i'll need this to update my UT for the build going out today, this is what we'll do for score = 0:
1. Euclidean dist: set it to max score and don't invert the distance.
2. Dot product: don't change the score, let it remain as is.